src/lib/y2storage/callbacks/commit.rb

```diff
@@ -21,61 +21,14 @@
 # To contact SUSE LLC about this file by physical or electronic mail, you may
 # find current contact information at www.suse.com.
 
-require "storage"
 require "yast"
```

Comment:
same here, why it is needed?

Reply:
Is needed for the `require "y2storage/callbacks/libstorage_callback"` below to be correct. The `require "yast"` changes the loading paths, so y2storage is found under `/usr/share/YaST/lib`.

Reply:
that is wrong, lib/y2storage/callbacks/commit.rb is already in special load path, so how you load it without modifyed LOAD_PATH?

Reply:
Fair enough, I will remove it.